ep/build-velox/src/get_velox.sh

```diff
@@ -106,15 +106,15 @@ function process_setup_ubuntu {
   fi
   sed -i "s/apt install -y/sudo apt install -y/" ${VELOX_HOME}/scripts/setup-adapters.sh
   if [ $ENABLE_S3 == "ON" ]; then
-    sed -i '/^  run_and_time install_folly/a \ \ '${VELOX_HOME}/scripts'/setup-adapters.sh aws' scripts/setup-ubuntu.sh
+    sed -i '/^  run_and_time install_folly/a \ \ export SUDO \n '${VELOX_HOME}/scripts'/setup-adapters.sh aws' scripts/setup-ubuntu.sh
```

Comment:
I cannot figure out the functionality of `export SUDO` here. Could you clarify? Thanks!

Reply:
In `scripts/setup-ubuntu.sh`, we have `SUDO="${SUDO:-"sudo --preserve-env"}"`.
`export SUDO` passes env `SUDO` to `scripts/setup-adapters.sh` so that `sudo` command is used to install aws/gcs/abfs.

Reply:
I guess the issue is related to this code:
https://github.com/facebookincubator/velox/blob/main/scripts/setup-helper-functions.sh#L156

Maybe, we can just use `SUDO="${SUDO:-"sudo --preserve-env"}"` to let "sudo" be used by default.